lib/src/core/mod.rs

```diff
@@ -0,0 +1,37 @@
+use anyhow::{anyhow, Context, Result};
+extern crate serde;
+use crate::*;
+use async_trait::async_trait;
+use serde_json::Value;
+
+mod proxy;
+pub mod verifier;
+
+#[async_trait]
+pub trait Verifier {
+    async fn evaluate(&self, evidence: &Value) -> Result<String>;
+}
+
+#[derive(Debug)]
+pub struct Core {
```

Comment:
`Core` is a rather vague name. Could we just call it `AttestationService` or something else a little bit more descriptive?